src/core/public/_css_variables.scss

```diff
@@ -7,8 +7,6 @@
   --kbnHeaderOffsetWithBanner: calc(var(--kbnHeaderBannerHeight) + var(--kbnHeaderOffset));
   // height of the action menu in the header in serverless projects
   --kbnProjectHeaderAppActionMenuHeight: #{$euiSize * 3};
-  // total height of all fixed headers for the app container in serverless projects, dynamically changes based on whether the banner or action menu is present
-  --kbnAppHeadersOffset: var(--euiFixedHeadersOffset, 0);
```

Comment:
Removing this from `:root` is a minor cleanup preference. IMO, this variable only needs to get set on the appropriate body modifier (since it's only used then). Other CSS can conditionally use either this variable or `--euiFixedHeadersOffset` if present by simply nesting var()` fallbacks:

https://github.com/Dosant/kibana/blob/9fc3c18ed1b3479cc28b2539e5c3b57b8347c749/src/core/public/styles/rendering/_base.scss#L22